commit 1a6ea24a0e7be9d12ae51c4de12124ad27c46ace (refs/changes/63/3463/1)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2018-02-23T15:19:23-08:00 (1 year, 7 months ago)
    
    TOOLS-1983 How to avoid hard failure of most builds on Feb 22 from GitHub weak crypto removal

diff --git a/targets.json.in b/targets.json.in
index 322e068..3c2c1b8 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -491,9 +491,9 @@ cat <<EOF
     "image_name": "ufds",
     "image_description": "SDC UFDS",
     "image_version": "1.0.0",
-    "image_uuid": "fd2cc906-8938-11e3-beab-4359c665ac99",
+    "// image_uuid": "triton-origin-multiarch-15.4.1@1.0.1",
+    "image_uuid": "04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f",
     "pkgsrc": [
-      "openldap-client-2.4.24",
       "postgresql91-client-9.1.2"
     ],
     "repos": [
